src/renderTemplate.js

```diff
@@ -0,0 +1,12 @@
+const RenderPosition = {
+  BEFOREBEGIN: 'beforebegin',
+  AFTERBEGIN: 'afterbegin',
+  BEFOREEND: 'beforeend',
+  AFTEREND: 'afterend',
+};
+
+const renderTemplate = (container, template, place) => {
+  container.insertAdjacentHTML(place, template);
+};
+
+export {RenderPosition, renderTemplate};
```

Comment:
сделай папку helpers или файлик helpers и храни вспомогательные функции там. 

```suggestion
export { RenderPosition, renderTemplate };
```

И пиши однообразно

Reply:
Исправил